lib/mini_sql/version.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 module MiniSql
-  VERSION = "1.1.3"
+  VERSION = "1.2.0"
```

Comment:
The build is failing on an error in the gem so I'm not sure if we should do a version bump until we can be sure that the gem itself isn't broken?

Reply:
yes... on it ... it seems I merged a problem PR a while back it forgot to require 'set'